docs/getting-started/products/ylockers/overview.md

```diff
@@ -1,8 +1,111 @@
 # Overview
-yLockers is a category of assets built by Yearn designed to tokenize locked governance positions in external DeFi protocols. This asset type is often called a "Liquid Locker".
 
-To achieve this, Yearn has deployed a system of smart contracts that allows users to permissionlessly max-lock their governance tokens to a central Yearn in exchange for a yLocker token (e.g., yCRV, yPRISMA, etc.) at a rate of 1:1. 
+:::info
+
+The new Ylockers ecosystem is rolling out on May 9th, 2024 with the launch of the *YearnBoostedStaker* (YBS) contract and a new V3 Auto-Compounder vault. These new contracts will be available for yPRISMA initially and then be rolled out for yCRV a month later on June 6th. Before June 6th, yCRV will continue to operate with the existing st-yCRV and lp-yCRV V2 vaults.
+
+:::
+
+yLockers are a category of assets built by Yearn designed to tokenize locked governance positions in external DeFi protocols. This asset type is often called a "Liquid Locker".
+
+To achieve this, Yearn has deployed a system of smart contracts that allows users to permissionlessly max-lock their governance tokens to a central Yearn in exchange for a yLocker token (e.g., yCRV, yPRISMA, etc.) at a rate of 1:1.
```

Comment:
`to a central Yearn`...probably add the work `contract` here?

Reply:
Ah, I thought I had changed this to `...governance tokens to Yearn in exchange for...`

Do you think `central Yearn contract` is better than simply `to Yearn`?

Reply:
tbh I think either is fine. and just `to Yearn` keeps it simpler

Reply:
updated to be `to Yearn`. marking as resolved